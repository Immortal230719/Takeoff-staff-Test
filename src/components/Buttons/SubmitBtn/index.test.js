import React from 'react';
import { shallow } from 'enzyme';
import { SubmitBtn } from './index';

describe('SubmitBtn', () => {
  const component = shallow(<SubmitBtn>Submit</SubmitBtn>);
  it('should render correctly', () => {

    expect(component.prop('children')).toEqual('Submit');
    expect(component).toMatchSnapshot();
  });
});
