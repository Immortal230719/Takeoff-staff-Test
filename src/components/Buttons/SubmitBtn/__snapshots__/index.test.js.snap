// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitBtn should render correctly 1`] = `
<WithStyles(ForwardRef(Button))
  className="makeStyles-btnSubmit-1"
  color="inherit"
  endIcon={
    <UNDEFINED
      color="inherit"
      fontSize="large"
    />
  }
  size="large"
  type="submit"
  variant="contained"
>
  Submit
</WithStyles(ForwardRef(Button))>
`;
